crates/agent-sql/src/publications.rs

```diff
@@ -208,6 +208,62 @@ pub async fn resolve_spec_rows(
     .await
 }
 
+#[derive(Debug, Serialize)]
+pub struct Tenant {
+    pub name: String,
+    pub tasks_quota: i32,
```

Comment:
nit: is there a reason for the i32 / i64 distinction here? Could they be `isize` ?

Reply:
Ummm... I tried a couple of different ways to get them to be the same and I thought sqlx's inference complained about each of them. It looks like making them all `i32` actually _does_ work now, so I'll go with that. FWIW `isize` does not work, sqlx doesn't know how to infer it correctly and complains that:

```
the trait bound `isize: sqlx::Decode<'_, Postgres>` is not satisfied
the following other types implement trait `sqlx::Decode<'r, DB>`:
  f32
  f64
  i16
  i32
  i64
  i8
```